src/Utils/Markdown.php

```diff
@@ -22,22 +24,18 @@
 class Markdown
 {
     private $parser;
-    private $purifier;
+    private $sanitizer;
 
-    public function __construct()
+    public function __construct(SanitizerInterface $sanitizer)
     {
         $this->parser = new \Parsedown();
```

Comment:
it's just a nail-care, but `Parsedown` could be moved to constructor too?

Reply:
I thought about it and thought it actually shouldn't be part of this PR, as it's not the same aim. I can definitely do another PR with a service for the parser though.